### PR TITLE
[DOC-137] updates to packed row from beta to Early access.

### PR DESCRIPTION
**Pull request by aishwarya24**
Build preview: https://deploy-preview-19395--infallible-bardeen-164bc9.netlify.app/preview/architecture/docdb/persistence/#packed-row-format

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 1]
```diff
@@ -646,7 +646,7 @@ Valid values: `SERIALIZABLE`, `REPEATABLE READ`, `READ COMMITTED`, and `READ UNC
 
 Default: `READ COMMITTED`<sup>$</sup>
 
-<sup>$</sup> Read Committed support is currently in [Tech Preview](/preview/releases/versioning/#feature-availability). Read Committed Isolation is supported only if the YB-TServer flag `yb_enable_read_committed_isolation` is set to `true`. By default this flag is `false` and in this case the Read Committed isolation level of the YugabyteDB transactional layer falls back to the stricter Snapshot Isolation (in which case `READ COMMITTED` and `READ UNCOMMITTED` of YSQL also in turn use Snapshot Isolation).
+<sup>$</sup> Read Committed Isolation is supported only if the YB-TServer flag `yb_enable_read_committed_isolation` is set to `true`. By default this flag is `false` and in this case the Read Committed isolation level of the YugabyteDB transactional layer falls back to the stricter Snapshot Isolation (in which case `READ COMMITTED` and `READ UNCOMMITTED` of YSQL also in turn use Snapshot Isolation).
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-tserver.md:649]: Are you including read committed documentation updates too?

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 2]
```diff
@@ -851,24 +851,6 @@ Rate control across all tablets being remote bootstrapped from or to this proces
 
 Default: `256MB` (256 MB/second)
 
-##### --remote_bootstrap_from_leader_only
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-tserver.md:854]: Why is this being removed?

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 3]
```diff
@@ -1013,6 +997,8 @@ Whether packed row is enabled for YSQL.
 
 Default: `false`
 
+For versions 2.16 and 2.18, the feature is stable to be used in production, and starting from version 2.19 and later, the flag default is true.
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-tserver.md]: true for new clusters.

**Review by rthallamko3 — CHANGES_REQUESTED** [review]
remove the unrelated changes.

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 4]
```diff
@@ -1003,7 +1003,9 @@ In addition, as this setting does not propagate to PostgreSQL, it is recommended
 --ysql_pg_conf_csv="ssl_min_protocol_version=TLSv1.2"
 ```
 
-## Packed row flags (Beta)
+## Packed row flags
+
+Packed row format support is currently in [Early Access](/preview/releases/versioning/#feature-availability).
 
 To learn about the packed row feature, see [Packed row format](../../../architecture/docdb/persistence/#packed-row-format-beta) in the architecture section.
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-tserver.md:1010]: This link needs to be updated in a couple of places. Can you search for #packed-row-format-beta and fix them all?

**Review by rthallamko3 — CHANGES_REQUESTED** [review]
need to fix a couple of links.

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 5]
```diff
@@ -1013,6 +1015,8 @@ Whether packed row is enabled for YSQL.
 
 Default: `false`
 
+For versions 2.16.4 and later, and 2.18.1 and later, the feature is stable to be used in production, and starting from version 2.19 and later, the flag default is true for new clusters.
```
Comment (rthallamko3) [inline docs/content/preview/reference/configuration/yb-tserver.md:1018]: Packed Row for YSQL can be used 2.16.4 release onwards in production environments, if the cluster is not used in xCluster settings. For xCluster scenarios, use 2.18.1 and later. Starting from version 2.19 and later, the flag default is true for new clusters.

File: docs/content/stable/reference/configuration/yb-tserver.md [hunk 6]
```diff
@@ -933,6 +935,8 @@ Whether packed row is enabled for YSQL.
 
 Default: `false`
 
+For versions 2.16.4 and later, and 2.18.1 and later, the feature is stable to be used in production, and starting from version 2.19 and later, the flag default is true for new clusters.
```
Comment (rthallamko3) [inline docs/content/stable/reference/configuration/yb-tserver.md:938]: Packed Row for YSQL can be used 2.16.4 release onwards in production environments, if the cluster is not used in xCluster settings. For xCluster scenarios, use 2.18.1 and later. Starting from version 2.19 and later, the flag default is true for new clusters.

**Review by rthallamko3 — CHANGES_REQUESTED** [review]

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 7]
```diff
@@ -1021,6 +1025,8 @@ Default: `0`
 
 ##### --ycql_enable_packed_row
 
+--ycql_enable_packed_row support is currently in [Tech Preview](/preview/releases/versioning/#feature-availability).
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:1028]:
```suggestion
YCQL packed row support is currently in [Tech Preview](/preview/releases/versioning/#feature-availability).
```

File: docs/content/stable/architecture/docdb/persistence.md [hunk 8]
```diff
@@ -235,7 +237,7 @@ operation only involves adding a delete marker at the correct level, and does no
 (hash1, 'user1', 20), msg_props_column_id, 'from', T3 -> 'c@d.com'
 (hash1, 'user1', 20), msg_props_column_id, 'subject', T3 -> 'bar'</code>
 </pre>
-The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout per se, as the writes happen in a log-structured manner.
+The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout by itself, as the writes happen in a log-structured manner.
```
Comment (ddhodge) [inline docs/content/stable/architecture/docdb/persistence.md:240]:
```suggestion
The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout, as the writes happen in a log-structured manner.
```

File: docs/content/preview/architecture/docdb/persistence.md [hunk 9]
```diff
@@ -237,7 +239,7 @@ operation only involves adding a delete marker at the correct level, and does no
 (hash1, 'user1', 20), msg_props_column_id, 'from', T3 -> 'c@d.com'
 (hash1, 'user1', 20), msg_props_column_id, 'subject', T3 -> 'bar'</code>
 </pre>
-The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout per se, as the writes happen in a log-structured manner.
+The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout by itself, as the writes happen in a log-structured manner.
```
Comment (ddhodge) [inline docs/content/preview/architecture/docdb/persistence.md:242]:
```suggestion
The key-value pairs that are displayed in strike-through font are logically deleted. The preceding DocDB layout is not the physical layout, as the writes happen in a log-structured manner.
```

**Review by ddhodge — APPROVED** [review]

**Review by rthallamko3 — APPROVED** [review]

**Pull request merged**